ceremonies/src/lib.rs

```diff
@@ -278,7 +278,6 @@ pub mod pallet {
 			let (cindex, meetup_index, meetup_participants, _meetup_location, _meetup_time) =
 				Self::gather_meetup_data(&cid, &sender)?;
 
-			ensure!(meetup_participants.contains(&sender), Error::<T>::OriginNotParticipant);
```

Comment:
this ensure is futile because the meetup is derived from the participant and the above line would fail if there a^was no assignment for the sender